contracts/InterchainTokenService.sol

```diff
@@ -304,6 +304,11 @@ contract InterchainTokenService is
 
         if (deployer == interchainTokenFactory) {
             deployer = TOKEN_FACTORY_DEPLOYER;
+        } else if (bytes(destinationChain).length == 0) {
```

Comment:
add dev comments on this new restriction

Reply:
updated.